tutorials/shaders/shader_reference/spatial_shader.rst

```diff
@@ -135,50 +135,58 @@ is usually:
-| inout float **POINT_SIZE**           | Point size for point rendering.                       |
-+--------------------------------------+-------------------------------------------------------+
-| in int **INSTANCE_ID**               | Instance ID for instancing.                           |
-+--------------------------------------+-------------------------------------------------------+
-| in vec4 **INSTANCE_CUSTOM**          | Instance custom data (for particles, mostly).         |
-+--------------------------------------+-------------------------------------------------------+
++--------------------------------------+--------------------------------------------------------+
+| Built-in                             | Description                                            |
++======================================+========================================================+
+| in vec2 **VIEWPORT_SIZE**            | Size of viewport (in pixels).                          |
++--------------------------------------+--------------------------------------------------------+
+| inout mat4 **WORLD_MATRIX**          | Model space to world space transform.                  |
++--------------------------------------+--------------------------------------------------------+
+| in mat4 **INV_CAMERA_MATRIX**        | World space to view space transform.                   |
++--------------------------------------+--------------------------------------------------------+
+| inout mat4 **PROJECTION_MATRIX**     | View space to clip space transform.                    |
++--------------------------------------+--------------------------------------------------------+
+| in mat4 **CAMERA_MATRIX**            | View space to world space transform.                   |
++--------------------------------------+--------------------------------------------------------+
+| inout mat4 **MODELVIEW_MATRIX**      |                                                        |
++--------------------------------------+--------------------------------------------------------+
+| inout mat4 **INV_PROJECTION_MATRIX** | Clip space to view space transform.                    |
++--------------------------------------+--------------------------------------------------------+
+| in vec3 **NODE_POSITION_WORLD**      | Node world space position.                             |
++--------------------------------------+--------------------------------------------------------+
+| in vec3 **NODE_POSITION_VIEW**       | Node view space position.                              |
++--------------------------------------+--------------------------------------------------------+
+| in vec3 **CAMERA_POSITION_WORLD**    | Camera world space position.                           |
++--------------------------------------+--------------------------------------------------------+
+| in vec3 **CAMERA_DIRECTION_WORLD**   | Camera world space direction.                          |
```

Comment:
I know these are the same as master, but I'm just noticing that re-arranging the order would be more clear in English. I.e. ``Node world space position.`` becomes ``Node position, in world space.``